CHANGELOG.md

```diff
@@ -2,8 +2,25 @@
 
 ## unreleased
 
+## 3.0.0-beta.1 / 2024-09-25
+
+* [CHANGE] regexp `.` now matches all characters (performance improvement). #14505
 * [CHANGE] `holt_winters` is now called `double_exponential_smoothing` and moves behind the [experimental-promql-functions feature flag](https://prometheus.io/docs/prometheus/latest/feature_flags/#experimental-promql-functions). #14930
+* [CHANGE] Prometheus will not add or remove port numbers from the target address. `no-default-scrape-port` feature flag removed. #14160
+* [ENHANCEMENT] UI: Many fixes and improvements. #14898, #14899, #14907, #14908, #14912, #14913, #14914, #14931, #14940, #14945, #14946, #14972, #14981, #14982, #14994
```

Comment:
Now we can add #15096 #15082 here.

Reply:
I put #15082 on a new line, as notifications seemed to be a notable enhancement. (is it a feature?)
Are there any other kinds of UI notifications?